airflow/cli/cli_parser.py

```diff
@@ -742,13 +722,15 @@ class ActionCommand(NamedTuple):
     help: str
     func: Callable
     args: Iterable[Arg]
+    description: str = ''
```

Comment:
```suggestion
    description: Optional[st]r = None
```
This makes mypy more aware of the types. Then it can suggest more.